packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -66,7 +66,9 @@ export class Persona extends React.Component<IPersonaProps, any> {
 
     let divProps = getNativeProps(this.props, divProperties);
     let personaDetails = <div className='ms-Persona-details'>
-      <div className='ms-Persona-primaryText'>{ primaryText }</div>
+      <div className='ms-Persona-primaryText'>
+        <div className='ms-Persona-primaryTextContent'>{ primaryText }</div>
```

Comment:
Ah, thought we needed this; I guess I didn't test it without the child div.  Removed in next commit